build.sh

```diff
@@ -649,11 +644,26 @@ case "$COMMAND" in
 
     "verify-cocoapods")
         pod setup
```

Comment:
Should do something like `git diff-index --quiet HEAD || ( echo "Cannot test the podspec with uncommitted changes"; exit 1)` since it does a hard reset.